src/qml/FeatureForm.qml

```diff
@@ -235,276 +235,292 @@ Page {
     id: fieldItem
 
     Item {
-      height: childrenRect.height
-      anchors {
-        left: parent ? parent.left : undefined
-        right: parent ? parent.right : undefined
-        leftMargin: 12
-      }
-
-      Item {
-        property string qmlCode: EditorWidgetCode !== undefined ? EditorWidgetCode : ''
-
-        id: qmlContainer
-        visible: Type == 'qml' && form.model.featureModel.modelMode != FeatureModel.MultiFeatureModel
-        height: visible ? childrenRect.height : 0
+        height: childrenRect.height
         anchors {
-          left: parent.left
-          right: parent.right
+          left: parent ? parent.left : undefined
+          right: parent ? parent.right : undefined
         }
 
-        Label {
-          id: qmlLabel
-          width: parent.width
-          text: Name || ''
-          wrapMode: Text.WordWrap
-          font.pointSize: 12
-          font.bold: true
-          bottomPadding: 5
-          color: 'grey'
+        // Configured color of the container
+        Rectangle {
+            color: Color
+            width: 8
+            height: parent.height
+            anchors.left: parent.left
         }
 
         Item {
-          id: qmlItem
           height: childrenRect.height
           anchors {
             left: parent.left
-            rightMargin: 12
             right: parent.right
```

Comment:
All looks good, just not sure if this is not going to cause warnings, as line 240 did?

Reply:
This one should always have an "Item" as parent. Do you know how this can be checked?

Reply:
When a component is used as a delegate, while redrawing the parent is set to `undefined` and causes a warning. I am pretty sure this is only for immeadiate parents of delegates like `id == fieldItem` component. So in this case there should not be a warning, just asking in case you saw something fishy in the logs with this branch during runtime in the logs.

Reply:
Makes sense.
I didnt see anything but also didn't pay attention...

Reply:
No warning here